doc/nrf/protocols/esb/index.rst

```diff
@@ -302,3 +302,26 @@ Examples
 The |NCS| provides the following example application that shows how to use the ESB protocol:
 
 * :ref:`esb_prx_ptx`
+
+.. _esb_fast_ramp_up:
+
+Fast ramp-up
+============
+
+The ESB driver is designed for bidirectional data transfer between devices.
+This implies that the radio peripheral is continuously switching between reception and transmission mode.
+When switching to transmission or reception mode, the radio peripheral needs some time to start internal analog components.
+On nRF51 and nRF24L series, a hardware-limited 130 µs delay is implemented.
+If ESB connection is established only between nRF52 and/or nRF53 devices, this delay can be reduced to 40 µs.
```

Comment:
```suggestion
If the ESB connection is established only between nRF52 or nRF53 Series devices, this delay can be reduced to 40 µs.
```

Reply:
Let's keep "end/or" to show that there are three possibilities: 52-52, 53-53, and 52-53.

Reply:
Done